### PR TITLE
Add autofixer to `no-unknown-arguments-for-builtin-components` rule

### PR DESCRIPTION
**Pull request by dwickern**
Before:
```hbs
<Input @value="1" @elementId="42" @disabled="disabled" />
<LinkTo @route="info" @model={{this.model}} @doubleClick={{action this.click}} />
<Textarea @value="1" @key-up={{this.onKeyUp}} />
```
After:
```hbs
<Input @value="1" id="42" disabled="disabled" />
<LinkTo @route="info" @model={{this.model}} {{on "dblclick" (action this.click)}} />
<Textarea @value="1" {{on "keyup" this.onKeyUp}} />
```

### CODE REVIEW

**Comment by rwjblue** [comment]
Would you mind regenerating the `README.md`?

File: lib/rules/no-unknown-arguments-for-builtin-components.js [hunk 1]
```diff
@@ -278,6 +279,31 @@ function removeAtSymbol(txt) {
...
 
+function fix(node, argument) {
+  const tagMeta = KnownArguments[node.tag];
+  const deprecatedArgs = tagMeta.deprecatedArguments || {};
+  const replacementArg = deprecatedArgs[argument.name];
+  if (replacementArg) {
+    argument.name = replacementArg;
+    return true;
+  }
+  const deprecatedEvents = tagMeta.deprecatedEvents || {};
```
Comment (bmish) [inline lib/rules/no-unknown-arguments-for-builtin-components.js:290]: Can you add a comment with an example conversion to each of the two parts of this `fix` function to make it clear to the reader what is going on?

**Review by bmish — APPROVED** [review]

File: lib/rules/no-unknown-arguments-for-builtin-components.js (hunk 1 above)
Comment (lifeart) [reply lib/rules/no-unknown-arguments-for-builtin-components.js:290]: Oh, sorry, lost in notifications and forgot about it :(

Comment (bmish) [reply lib/rules/no-unknown-arguments-for-builtin-components.js:290]: No problem, feel free to follow-up with improvements.

**Pull request merged**